components/launcher/src/lib.rs

```diff
@@ -12,3 +12,4 @@ mod sys;
 
 pub const SUP_CMD: &str = "hab-sup";
 pub const SUP_PACKAGE_IDENT: &str = "core/hab-sup";
+pub const VERSION: Option<&str> = option_env!("PLAN_VERSION");
```

Comment:
The launcher actually uses a different versioning scheme than most of our other components. It does not use the version in the `VERSION` file but instead does a count of git commits at build time in the habitat plan. So in the launcher's case, a plain local cargo build will not have a version.

Reply:
Ah, that's right... I wonder if it would make sense to make this be `u32::max` or something in the case that `PLAN_VERSION` is not set. It seems probable that we will have launcher version checks that check for some minimum version. Having local builds always be 0 could cause features to not work that should.